README.md

```diff
@@ -19,3 +19,9 @@ Supported VS Code settings
 The following settings are supported:
 
 * `vividus-studio.java-home`: The directory path containing JDK to run the extension, if not specified the JDK is located automatically.
+
+Supported VS Code commands
+==========================
+The following commands are supported:
+
+* `VIVIDUS: Run Stories`: The command executes configured VIVIDUS stories.
```

Comment:
it doesn't bring any value, too obvious for me to add some extra docs

Reply:
for me everything in https://docs.vividus.dev/vividus/latest/getting-started.html, lets remove it since it brings 0 value to me?